### ISSUE TITLE
Prisma Migrate fails to load .env if non-standard name is used for schema file

### ISSUE THREAD

**Issue by siberian-man** [prisma/migrate#483]
#  Bug description

This command fails (schema file is named as xxx.prisma):
```
npx prisma migrate save --name '' --experimental --schema=./xxx.prisma
```

while this one works fine (schema file is named as schema.prisma):
```
npx prisma migrate save --name '' --experimental
```

## How to reproduce

Just use the aforementioned commands.

## Expected behavior

Migration should not fail.

## Prisma information

Error log:

```
Error: Get config {"is_panic":false,"message":"error: Environment variable not found: DATABASE_URL.\n  -->  schema.prisma:3\n   | \n 2 |   provider = \"postgresql\"\n 3 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1","meta":{"full_error":"error: Environment variable not found: DATABASE_URL.\n  -->  schema.prisma:3\n   | \n 2 |   provider = \"postgresql\"\n 3 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1"},"error_code":"P1012"}
```

## Environment & setup

- OS: Mac
- Database: PostgreSQL
- Prisma version: 2.0.0-beta.5

**Comment by janpio** [comment]
Can you maybe update to a more recent version of Prisma? Might be that this is already fixed in 2.0.1.

**Comment by pantharshit00** [comment]
Closing as a duplicate of https://github.com/prisma/prisma/issues/2735#event-3482289910. This is fixed in dev channel
